Jenkinsfile

```diff
@@ -192,8 +206,31 @@ jobWrappers {
   }
 
   stage('Testing') {
-
     def branches = [:]
+    for (def d in ['derby','postgres']) {
```

Comment:
Do we plan to add mysql and oracle in a separate jira?

Reply:
definetly!...and also run the verification tests as well